src/firefox/index.js

```diff
@@ -251,7 +251,37 @@ export async function useProfile(
     customPrefs = {},
   }: UseProfileParams = {},
 ): Promise<FirefoxProfile> {
-  const profile = new FirefoxProfile({destinationDirectory: profilePath});
+  let profile;
+  const finder = new FirefoxProfile.Finder();
```

Comment:
we will need to be able to override the path where the profiles will be searched by the profile Finder helper class, to be able to more easily test these behaviors in the unit tests, and so this function should take an additional `searchProfilePath` (or another similar identifier name) parameter which will default to undefined (so that the default behavior of the Finder class will be preserved when this method is executed without passing the search profile path explicitly).